src/streams/EditStreamScreen.js

```diff
@@ -43,7 +58,7 @@ export default function EditStreamScreen(props: Props): Node {
         initialValues={{
           name: stream.name,
           description: stream.description,
-          privacy: stream.invite_only ? 'private' : 'public',
+          privacy: streamPropsToPrivacy(stream),
```

Comment:
EditStreamScreen [nfc]: Use streamPropsToPrivacy to convert initial value
    
    This will save quite a bit of duplication; see previous commit.

I don't understand the last reference. The previous commit is:
181bf22b3 EditStreamCard [nfc]: Use params object for PropsCreateStream, too

Ah, maybe you meant to refer to this one?
d2d63b595 streamsActions: Have updateExistingStream translate Privacy value for API

    The translation is about to get much more complicated: `Privacy`
    will have four branches, corresponding to the four stream-access
    policy settings:

Reply:
(The actual change seems good, though.)

Reply:
Yep! Thanks for catching this.